FirebaseAuth/Sources/MultiFactor/FIRMultiFactor.m

```diff
@@ -149,7 +147,7 @@ - (instancetype)initWithMFAEnrollments:(NSArray<FIRAuthProtoMFAEnrollment *> *)M
   if (self) {
     NSMutableArray<FIRMultiFactorInfo *> *multiFactorInfoArray = [[NSMutableArray alloc] init];
     for (FIRAuthProtoMFAEnrollment *MFAEnrollment in MFAEnrollments) {
-      if (MFAEnrollment.phoneInfo) {
+      if (MFAEnrollment.MFAValue) {
```

Comment:
Looks like a change in logic?

Reply:
Ah, I see what happened–– if you cmd + F for `phoneInfo` in https://github.com/firebase/firebase-ios-sdk/pull/10696/files, you can see that the name was changed to `MFAValue` when porting to Swift. I reverted it back to `phoneInfo` for now.